app/src/main/java/com/palone/planahead/screens/home/ui/components/ChooseTaskType/ChooseTaskType.kt

```diff
@@ -0,0 +1,47 @@
+package com.palone.planahead.screens.home.ui.components.ChooseTaskType
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.palone.planahead.data.database.alert.properties.TaskType
+
+@Composable
+fun ChooseTaskType(
```

Comment:
I think this could be a generic component, completely unrelated to tasks. Then we just use it with a map of value->label and get value in onValueChange. That way we would be able to use it for other types (e.g. alert trigger type).